Week1/homework/js-exercises/ex10_compareArrays.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+let arrMixed = [22.5, "Hello", true, { color: "blue" }];
+let arrNewWonders = ["Taj Mahal", "Colloseum", "Machu Picchu", "Christ the Redeemer", "Chichen Itza", "Petra", "Great Wall of China"];
+console.log("The length of the array arrMixed is: " + arrMixed.length);
+console.log("The length of the array arrNewWonders is: " + arrNewWonders.length);
+
+(arrMixed.length == arrNewWonders.length) ? console.log("They are the same!") : console.log("Two different sizes");
```

Comment:
```suggestion
message = arrMixed.length === arrNewWonders.length ? "They are the same!" : "Two different sizes";
console.log(message)
```

Slightly more readable adjustion ☝️ 
Also, always use [strict comparison](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality#:~:text=The%20strict%20equality%20operators%20(%20%3D%3D%3D,Algorithm%20to%20compare%20two%20operands.&text=If%20both%20operands%20are%20objects,operands%20are%20undefined%20%2C%20return%20true%20.)